src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -115,7 +115,7 @@ def _print_extensions_msg(help_file):
         if isinstance(help_file.command_source, ExtensionCommandSource):
             logger.warning(help_file.command_source.get_command_warn_msg())
             if help_file.command_source.preview:
-                logger.warning(help_file.command_source.get_preview_warn_msg())
+                logger.warning(help_file.command_source.get_preview_experimental_warn_msg())
```

Comment:
a little bit confused by this line.

Reply:
This function previously only gets the preview warning message. Now it gets both. Just renaming it to reflect the real functionality.

Reply:
but the condition is `if help_file.command_source.preview:`. Did I miss something?

Reply:
My fault. Will fix it.